crates/ark/src/modules/rstudio/commands.R

```diff
@@ -6,7 +6,14 @@
         "activateTerminal" = "workbench.action.terminal.focus",
         # This command includes untitled files in RStudio:
         "saveAllSourceDocs" = "workbench.action.files.saveAll",
-        "vcsRefresh" = "git.refresh",
+        # This command is a silent no-op in RStudio when there is no git repo:
+        "vcsRefresh" = {
+            if (.ps.ui.evaluateWhenClause("gitOpenRepositoryCount > 0")) {
```

Comment:
There is a lot of things you can check about the git situation (for example, [see what is checked to show the "The folder currently open doesn't have a Git repository. You can initialize..." screen](https://github.com/posit-dev/positron/blob/693987748002d8aed55c2913dbe58290a249cb10/extensions/git/package.json#L3357)) but I am pretty sure `gitOpenRepositoryCount > 0` is necessary + sufficient for the refreshing.

Reply:
I would not be against doing `config.git.enabled && gitOpenRepositoryCount > 0`, but I'll leave that up to you